### PR TITLE
Fix/photoatomic

### PR DESCRIPTION
**Pull request by whaeck**
Changes were made to the ACER routines for photoatomic data to fix an infinite loop when using ENDF/B-VIII.0 data and to fix the scratch array size to accommodate the current size of the ENDF data.

This fixes issues #91 and #135, see the issues for more information on the actual causes.

The release notes branch has been pulled into this one as well (pull request #134 will become obsolete when we pull this one into master).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,12 @@
 # NJOY2016
-[![Build Status](https://travis-ci.org/njoy/NJOY2016.svg?branch=master)](https://travis-ci.org/njoy/NJOY2016)
```
Comment (jlconlin) [inline README.md]: 👍

File: docs/testDescription.md [hunk 2]
```diff
@@ -246,3 +246,9 @@ title: NJOY2016 Test Descriptions
 
   This test is added following a change in `GROUPR` for the MF6 MT18 part of the GENDF file. Versions prior to NJOY 2016.49 only gave the infinite dilute fission matrix, independent of the number of sigma0 values requested by the user. This test runs two `GROUPR` runs, one with only infinite dilute and another one with two sigma0 values (including infinite dilute). The test also contains `ERRORR` runs for MF35 covariances to verify that the `ERRORR` module still gives the same results (only infinite dilute data is used) when using either of the produced GENDF files.
 
+## Test Problem 48
+
+[[input](https://raw.githubusercontent.com/njoy/NJOY2016/master/tests/48/input)]
+
+  This test is added following issues in processing photoatomic data in `ACER` (see issues #91 and #135). The problems were caused by truncation to 7 significant digits (knowing that the ENDF/B-VIII.0 files often go to 9 significant digits) and array sizes that were too small for the current data. This test was added to detect this problem in the future.
```
Comment (jlconlin) [inline docs/testDescription.md]: Do you want to make the issues links? I don't really care one way or the other.

File: src/vers.f90 [hunk 3]
```diff
@@ -3,7 +3,7 @@ module version
    ! These values are updated during the NJOY revision-control process.
    implicit none
    private
-   character(8),public::vers='2016.49'
-   character(8),public::vday='25Jan19'
+   character(8),public::vers='2016.50'
```
Comment (jlconlin) [inline src/vers.f90]: Nice job remembering this!

File: src/acepa.f90 [hunk 4]
```diff
@@ -514,7 +518,7 @@ subroutine alax(nin,nlax,matd,fluor,ener,abs,heat,a,nes)
    real(kr)::a(*)
    ! internals
    integer::loc(50)
-   integer::iz,nw,nb,nss,ll,iss,ntr,kk,idis,i,jj,n,mm
```
[the remark this answers is not in the document]
Comment (whaeck) [reply src/acepa.f90]: Unused variable.

File: docs/testDescription.md (hunk 2 above)
Comment (whaeck) [reply docs/testDescription.md]: I thought it was done automatically, apparently not.

**Comment by whaeck** [comment]
OK, just waiting on @kahlerac to see if he has issues/questions/... on this.

**Pull request merged**